sevntu-checks/sevntu-checks.xml

```diff
@@ -159,6 +159,14 @@
     <module name="com.github.sevntu.checkstyle.checks.coding.WhitespaceBeforeArrayInitializerCheck"/>
     <module name="com.github.sevntu.checkstyle.checks.coding.MoveVariableInsideIfCheck" />
     <module name="com.github.sevntu.checkstyle.checks.coding.RequireFailForTryCatchInJunitCheck" />
+    <module name="com.github.sevntu.checkstyle.checks.coding.Jsr305AnnotationsCheck">
```

Comment:
please add comment above
`<!-- we ignore this Check only because we are not ready to follow it and refactor whole project for now-->`

Reply:
Done with the next commit (I am still aggregating changes).